public/scripts/PromptManager.js

```diff
@@ -1550,22 +1550,53 @@ class PromptManager {
             const isInjectionPrompt = prompt.injection_position === INJECTION_POSITION.ABSOLUTE;
             const isOverriddenPrompt = Array.isArray(this.overriddenPrompts) && this.overriddenPrompts.includes(prompt.identifier);
             const importantClass = isImportantPrompt ? `${prefix}prompt_manager_important` : '';
+
+            //add role icons to the right of prompt name
+
+            const hasRole = prompt.role ? true : false;
+            let roleIcon, roleTitle;
+            if (hasRole) {
+                switch (prompt.role) {
+                    case 'system':
+                        roleIcon = 'fa-cog';
+                        roleTitle = 'Prompt will be sent as System';
+                        break;
+                    case 'assistant':
+                        roleIcon = 'fa-robot';
+                        roleTitle = 'Prompt will be sent as Assistant';
+                        break;
+                    case 'user':
+                        roleIcon = 'fa-user';
+                        roleTitle = 'Prompt will be sent as User';
+                }
```

Comment:
Could this be made into a map? Like:
```js
{
  system: {roleIcon: 'fa-cog',  roleTitle: '...'},
  assistant: {roleIcon: 'fa-robot',  roleTitle: '...'},
  user: {roleIcon: 'fa-user',  roleTitle: '...'}
}
```

(we can still store `roleIcon` and `roleTitle` as variables in case the code below needs to be uncommented)